components/QuizCardPage/Timeline/index.test.js

```diff
@@ -0,0 +1,13 @@
+import { render, screen } from "@testing-library/react";
+import QuizCard from "../../../pages/QuizCard";
+
+describe("StyledTimer Component", () => {
```

Comment:
You are testing `Timeline` / `StyledTimer` according to the description, but you are rendering the entire page. That's too much: simply import the Timer you want to test.